game/factions/faction.py

```diff
@@ -167,7 +167,9 @@ def from_json(cls: Type[Faction], json: Dict[str, Any]) -> Faction:
         faction.awacs = load_all_aircraft(json.get("awacs", []))
         faction.tankers = load_all_aircraft(json.get("tankers", []))
 
-        faction.aircrafts = list(set(faction.aircrafts + faction.awacs))
+        faction.aircrafts = list(
+            set(faction.aircrafts + faction.awacs + faction.tankers)
+        )
```

Comment:
I don't think there's actually any reason for these to be separate lists. AWACS is just for the sake of compatibility with older faction definitions, but I don't think it actually needs to be. Let's just add the tankers to the regular aircraft list and let the aircraft compat lists sort it out.

Reply:
(Never mind, I thought this was new but it isn't)